package.json

```diff
@@ -18,5 +18,10 @@
     "semi": true,
     "singleQuote": true,
     "quoteProps": "consistent"
-  }
+  },
+  "directories": {
+    "lib": "lib"
+  },
+  "main": "lib/range.js",
+  "type": "module"
```

Comment:
@dpup These lines are necessary to get https://github.com/range-labs/range-export working correctly. And judging from the version change above to 1.4.0, I think this was published to npm for the past year.